bpf/process/pfilter.h

```diff
@@ -513,6 +650,27 @@ selector_process_filter(__u32 *f, __u32 index, struct execve_map_value *enter,
 	if (res == PFILTER_REJECT)
 		return res;
 
+	/* matchCurrentCred
+	 *
+	 * Depends on LARGE_BPF_PROG to be available, if not then userspace writes
+	 * 4 bytes size length and advances the offset index with same 4 bytes.
+	 *
+	 * (4 (length) + 4 (flags) + sizeof(ruid) + sizeof(rgid) + ...
+	 */
+	len = *(__u32 *)((__u64)f + (index & INDEX_MASK));
+	index += 4; /* 4: matchCurrentCred header */
+
+#ifdef __LARGE_BPF_PROG
+	/* Filter current task credentials */
+	if (len > 4) {
+		len -= 4;
```

Comment:
should we follow the rest of the selectors and have the len -= 4 before the if condition and the condition have if (len > 0) ?